src/coreclr/jit/instr.h

```diff
@@ -490,7 +491,8 @@ enum emitAttr : unsigned
 #define EA_IS_GCREF_OR_BYREF(x)     ((((unsigned)(x)) & ((unsigned)(EA_BYREF_FLG | EA_GCREF_FLG))) != 0)
 #define EA_IS_DSP_RELOC(x)          ((((unsigned)(x)) & ((unsigned)EA_DSP_RELOC_FLG)) != 0)
 #define EA_IS_CNS_RELOC(x)          ((((unsigned)(x)) & ((unsigned)EA_CNS_RELOC_FLG)) != 0)
-#define EA_IS_RELOC(x)              (EA_IS_DSP_RELOC(x) || EA_IS_CNS_RELOC(x))
+#define EA_IS_CNS_SEC_RELOC(x)          ((((unsigned)(x)) & ((unsigned)EA_CNS_SEC_RELOC)) != 0)
+#define EA_IS_RELOC(x)              ((EA_IS_DSP_RELOC(x) || EA_IS_CNS_RELOC(x)) && !EA_IS_CNS_SEC_RELOC(x))
```

Comment:
why does the definition of `EA_IS_RELOC` need to change at all?

Reply:
Maybe there should be a comment about why DSP and CNS relocs are different from SEC relocs, and why `EA_IS_RELOC` should *not* include SEC relocs.